code/modules/cm_preds/yaut_items.dm

```diff
@@ -493,7 +493,6 @@
 
 		var/obj/item/weapon/gun/energy/plasma_caster/W = new(usr)
 		usr.put_in_active_hand(W)
-		W.source = src
```

Comment:
why

Reply:
I reworked the energy weapons code to use their own energy sources instead of the badly coded pred bracer one.